ClimbingGearBackend/Models/RackUsers.cs

```diff
@@ -0,0 +1,27 @@
+using Microsoft.EntityFrameworkCore;
+using System.Collections.Generic;
+
+namespace ClimbingGearBackend.Models
+{
+  [Keyless]
```

Comment:
What does this attribute do?

Reply:
It tells EF core to not generate primary keys for each entry. If you don't use that annotation EF core will still create a primary key field in the background.